src/renderer/global.scss

```diff
@@ -0,0 +1,145 @@
+/* :not(.chakra-slider__track):not(.chakra-menu__menu-button):not(.react-flow__handle)
+:not(.chakra-slider__track):not(.chakra-menu__menu-button):not(.react-flow__handle)
```

Comment:
Can you put the `.chakra-slider__track` rule back? This was there because without it, it makes the slider track not have the pointer hand, even though the surrounding area has the pointer hand.

Reply:
This was already like this before, I didn't comment it out. I only grouped selectors, so no functional changes were made.

Reply:
Oh..... Then why tf did I get rid of that?? Lol. Guess it's fine fine